packages/agent/src/framework-mounter.ts

```diff
@@ -8,7 +8,8 @@ import path from 'path';
 import { HttpCallback } from './types';
 
 export default class FrameworkMounter {
-  private readonly onStart: ((router: Router) => Promise<void>)[] = [];
+  private readonly onStartServer: (() => Promise<void>)[] = [];
```

Comment:
```suggestion
  private readonly onServerStart: (() => Promise<void>)[] = [];
```

Maybe it makes more sense.